### ISSUE TITLE
How To Contribute and guidelines

### ISSUE THREAD

**Issue by vegetable** [BSData/The-9th-Age#149]
I know that maybe this is not the place.

I'd contribute fixing/checking the catalogues, I tried solving:

https://github.com/BSData/The-9th-Age/issues/144

With no success :(

I read the doc's from README.md

So it would be great to have a document about how to edit the data files, and how you are doing it (for instance, how to set percentage limits, how to offer/deny some items, and so)

Thanks to all community for serving game data!

**Comment by Karanadon** [comment]
I'm pretty sure I already solved #144 so if it is still happening maybe refresh your data? The issue was decreasing the dark gods might character category limit for points spent in dark gods might and vice versa, there only needed to be one decrease, in just the dark gods might category. If you go to the main Battlescribe homepage there is a reasonable FAQ to see how to edit files. The way I learned was by doing things, and looking at what others had done before. Try making up a new army from scratch and you will sure learn very fast.
